deploy/lib/xquery/setup.xqy

```diff
@@ -5530,6 +5560,19 @@ declare function setup:get-user-id($user-name as xs:string) as xs:unsignedLong?
      </options>)
 };
 
+(: Gets the role names from the default security database :)
+declare function setup:get-existing-role-names() as element(sec:role-name)* {
+  let $role-names :=
+    xdmp:eval(
+        'import module namespace sec="http://marklogic.com/xdmp/security" at "/MarkLogic/security.xqy";
```

Comment:
indentation should be 2 spaces, so the quote here should be in column 6, as should the () a couple lines down and the &lt;options&gt;.